var/spack/repos/builtin/packages/hdf5/package.py

```diff
@@ -79,12 +79,12 @@ class Hdf5(AutotoolsPackage):
 
     @property
     def libs(self):
-        """Hdf5 can be queried for the following parameters:
+        """HDF5 can be queried for the following parameters:
 
         - "hl": high-level interface
         - "cxx": C++ APIs
-        - "fortran": fortran APIs
-
+        - "fortran": Fortran APIs
+        
```

Comment:
I'm excited to see whether or not flake8 catches this. I recently fixed a bug that was letting trailing whitespace in.